patches/node/build_add_gn_build_files.patch

```diff
@@ -1,7 +1,7 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Jeremy Apthorp <nornagon@nornagon.net>
 Date: Tue, 26 Feb 2019 17:07:45 -0800
-Subject: build: add GN build files
+Subject: build_add_GN_build_files.patch
```

Comment:
this doesn't seem right

Reply:
our chromium patch subjects are all the names of the patchfiles themselves; this was intended to bring our node patchfiles into a more consistent state with other patches